src/command/root-command/index.ts

```diff
@@ -71,4 +84,12 @@ export class RootCommand {
       }
     }
   }
+
+  private localRemoteUrlMismatch(): boolean {
```

Comment:
please add an other name to this function that reflects the role of this function and also add some jsdoc because it won't be straightforward later.